Makefile

```diff
@@ -22,12 +21,24 @@ run-api: build/keppel
 copy-fixtures:
 	find -name '*.actual' | xargs -I{} bash -c 'mv {} $$(echo {} | sed "s/.actual//g")'
 
-build-all: build/keppel
+privkey.pem:
+	openssl genrsa -out privkey.pem 4096
```

Comment:
Since this private key is specifically used for `run-oci-conformance`, the name should clarify that. I suggest putting both this file and also `.env-oci-conformance` into a separate subdirectory, e.g.

```
testing/oci-conformance/privkey.pem
testing/oci-conformance/env.sh
```